addons/sourcemod/scripting/shop.sp

```diff
@@ -1670,6 +1670,14 @@ bool GiveItemEx(int client, const char[] sItemId)
 	{
 		return false;
 	}
+	
+	if (value != 0)
+	{
+		if (type == Item_Finite)
+			count = value;
+		else
+			duration = value;
```

Comment:
and here

Reply:
Sorry, guys, about the brackets. I fixed that too. And I checked, there are no more such problems on my side.